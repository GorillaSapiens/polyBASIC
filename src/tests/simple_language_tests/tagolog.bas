10 PARASA x = 1 SA 10
20 IPRINT "hello world, or ðŸ‘‹ ðŸŒŽ!", x
30 SUSUNOD x
40 MAGTAPOS
