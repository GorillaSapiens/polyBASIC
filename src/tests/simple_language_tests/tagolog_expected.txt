hello world, or 👋 🌎! 1
hello world, or 👋 🌎! 2
hello world, or 👋 🌎! 3
hello world, or 👋 🌎! 4
hello world, or 👋 🌎! 5
hello world, or 👋 🌎! 6
hello world, or 👋 🌎! 7
hello world, or 👋 🌎! 8
hello world, or 👋 🌎! 9
hello world, or 👋 🌎! 10
